be/src/exec/pipeline/sort/local_partition_topn_context.h

```diff
@@ -68,6 +68,10 @@ class LocalPartitionTopnContext {
     // Pull one chunk from sorters or passthrough_buffer
     StatusOr<ChunkPtr> pull_one_chunk();
 
+    bool is_passthrough() const { return _chunks_partitioner->is_passthrough(); }
+
+    size_t num_partitions() const { return _partition_num; }
```

Comment:
`_partition_num` is already set before call this ?

Reply:
Yes, I'll call it after it is set.